TombEngine/Scripting/Internal/TEN/Strings/DisplayString/DisplayString.cpp

```diff
@@ -144,13 +144,19 @@ void DisplayString::Register(sol::table& parent)
 		// @tparam table table the new table with display flags options
 		// @usage
 		// local varDisplayString = DisplayString('example string', 0, 0, Color(255, 255, 255), false)
+		// possible values:
 		// varDisplayString:SetFlags({})
 		// varDisplayString:SetFlags({ TEN.Strings.DisplayStringOption.SHADOW })
 		// varDisplayString:SetFlags({ TEN.Strings.DisplayStringOption.CENTER })
 		// varDisplayString:SetFlags({ TEN.Strings.DisplayStringOption.SHADOW, TEN.Strings.DisplayStringOption.CENTER })
 		// -- When passing a table to a function, you can omit the parentheses
 		// varDisplayString:SetFlags{ TEN.Strings.DisplayStringOption.CENTER }
-		ScriptReserved_SetFlags, &DisplayString::SetFlags
+		ScriptReserved_SetFlags, &DisplayString::SetFlags,
+
+		//Set translated parameter of the string
+		// @tparam bool if true this will be the string key for the translation that will be displayed.
+		// If false, this will be the string that's displayed
+		ScriptReserved_SetTranslated, & DisplayString::SetTranslated
```

Comment:
`@tparam` should be `@tparam bool name`, where name is the name of the argument in the documentation, e.g. `@tparam bool shouldTranslate`. At the moment it thinks `if` is the name of the arg